chatmm/src/main/java/com/likeminds/chatmm/search/view/LMChatSearchFragment.kt

```diff
@@ -47,10 +53,25 @@ class LMChatSearchFragment : BaseFragment<LmChatFragmentSearchBinding, SearchVie
         }
     }
 
+    @RequiresApi(Build.VERSION_CODES.N)
+    override fun receiveExtras() {
+        super.receiveExtras()
+        if (arguments == null || arguments?.containsKey(LM_CHAT_SEARCH_EXTRAS) == false) {
+            requireActivity().supportFragmentManager.popBackStack()
+            return
+        }
+
+        searchExtras = ExtrasUtil.getParcelable(
+            requireArguments(),
+            LM_CHAT_SEARCH_EXTRAS,
+            LMChatSearchExtras::class.java
+        ) ?: return
```

Comment:
in return throw `emptyExtraException()`

Reply:
Not throwing exception as it is not required in all cases